web/src/containers/IncentivePayments.js

```diff
@@ -31,69 +36,65 @@ class IncentivePayments extends Component {
+            <h3 id={thId(year)}>{t('ffy', { year })}</h3>
+            <table className="table-cms table-fixed">
+              <thead>
+                <tr>
+                  <th id={thId('null2')} />
+                  <Fragment key={year}>
+                    {QUARTERS.map(q => (
+                      <th key={q} className="right-align" id={thId(year, q)}>
+                        {t('table.quarter', { q })}
+                      </th>
+                    ))}
+                    <th className="right-align" id={thId(year, 'subtotal')}>
+                      {t('table.subtotal')}
                     </th>
-                  ))}
-                  <th
-                    className="right-align"
-                    id={thId(year, 'subtotal')}
-                  >
-                    {t('table.subtotal')}
-                  </th>
-                </Fragment>
-              </tr>
-            </thead>
-            <tbody>
-              {INCENTIVE_ENTRIES.map(({ id, name, type }, i) => {
-                const InputComponent = type === 'amount' ? DollarInput : Input;
-
-                return (
-                  <tr key={id}>
-                    <td
-                      className={`align-middle ${i % 2 === 0 ? 'bold' : ''}`}
-                      headers="incentive-payments-table-fynull1 incentive-payments-table-fynull2"
-                    >
-                      {name}
-                    </td>
-                    <Fragment key={year}>
-                      {QUARTERS.map(q => (
-                        <td key={q} headers={tdHdrs(year, q)}>
-                          <InputComponent
-                            name={`${id}-payments-${year}-q${q}`}
-                            label={`${id} payments for ${year}, quarter ${q}`}
-                            wrapperClass="m0"
-                            className="m0 input input-condensed mono right-align"
-                            value={data[id][year][q] || ''}
-                            onChange={this.handleChange(id, year, q)}
-                            hideLabel
-                          />
-                        </td>
-                      ))}
+                  </Fragment>
+                </tr>
+              </thead>
+              <tbody>
+                {INCENTIVE_ENTRIES.map(({ id, name, type }, i) => {
+                  const InputComponent =
+                    type === 'amount' ? DollarInput : Input;
+
+                  return (
+                    <tr key={id}>
                       <td
-                        className="bold mono right-align align-middle"
-                        headers={tdHdrs(year, 'subtotal')}
+                        className={`align-middle ${i % 2 === 0 ? 'bold' : ''}`}
+                        headers="incentive-payments-table-fynull1 incentive-payments-table-fynull2"
                       >
-                        {formatNumber(type, totals[id].byYear[year])}
+                        {name}
```

Comment:
I'm not sure what these changes are.  I think they came from another PR but then I don't know why they'd show up like this here.  I didn't do it, though!  😬

Reply:
welp... they seem fairly benign, at least? 🤷‍♀️